cmd/aries-agent-mobile/pkg/wrappers/config/options.go

```diff
@@ -23,6 +25,8 @@ type Options struct {
 	// not intended to be used by golang code
 	HTTPResolvers     []string
 	OutboundTransport []string
+
+	Notifier api.Notifier
```

Comment:
the interface should be `command.Notifier`

Reply:
and we need one more implementation. It will be used for SDK (for REST we will use web sockets )